src/test/regress/expected/window_functions.out

```diff
@@ -1386,12 +1386,16 @@ LIMIT 5;
                                  Sort Key: users_table.user_id, (avg(users_table.value_1)) DESC
```

Comment:
Can we avoid the alternative output by not having the whole explain plans and have simple helpers (like `WindowAggIsPushedDown` / `WindowAggIsPulledUp` / `LimitPushedDown`)? Because all we care seems those type of checks?

Reply:
This didn't seem easy as we talked offline